src/ElasticsearchDAO.py

```diff
@@ -7,17 +7,17 @@ class ElasticsearchDAO(object):
     def __init__(self):
         super().__init__()
 
-    def writeToElasticsearch(self, datasets):
+    def write_To_Elasticsearch(self, datasets):
```

Comment:
The variable renames on this page will not pass the [sonar code smell](https://sonarcloud.io/project/issues?id=usdot-its-jpo-data-portal_datahub-ingest&open=AXE8FdPXOQUa5KFegj3H&resolved=false&types=CODE_SMELL) you are aiming to fix.

The variable renames will need to change to snake case where there's an underscore separating each word and all letters are lowercased. For example:
- writeToElasticsearch --> write_to_elasticsearch
- existingDocs --> existing_docs